requirements.txt

```diff
@@ -43,9 +43,12 @@ nocasedict>=1.0.1
 # urllib3 1.25.9 addressed issue 38834 reported by safety
 # urllib3 needs to be pinned to <1.25 for requests <2.22.0
 # urllib3 1.26.5 vendors six 1.16.0 which is needed on Python 3.10 to remove ImportWarning
-urllib3>=1.25.9; python_version == '2.7'
-urllib3>=1.25.9; python_version >= '3.5' and python_version <= '3.9'
-urllib3>=1.26.5; python_version >= '3.10'
+# urllib3 2.0 requires py>=3.7
+urllib3>=1.25.9,<2.0.0; python_version >= '2.7' and python_version <= '3.6'
+# TODO: The following limits use of urllib <= 2.0 until issues about this upgrade
+# resolved. Issue #3006 filed to update urllib3 to version 2.0.0.
+urllib3>=1.25.9,<2.0.0; python_version >= '3.7' and python_version <= '3.9'
```

Comment:
I don't understand the separation between the python versions 2.7-3.6 and 3.7-3.9. The Todo comment refers to the following lines, but also applies to the previous line.

Can we merge the two sets of Python versions?